tests/integration/targets/anycast_config_info/tasks/main.yml

```
@@ -0,0 +1,70 @@
+---
+- module_defaults:
+    group/infoblox.universal_ddi.all:
+      portal_url: "{{ portal_url }}"
+      portal_key: "{{ portal_key }}"
+  block:
+    - ansible.builtin.set_fact:
+        ac_config_name: "test_ac_config-{{ 99999 | random }}"
+        tag_value: "site-{{ 999999 | random | string }}"
+
+    - name: Create Anycast Configuration
+      infoblox.universal_ddi.anycast_config:
+        name: "{{ ac_config_name }}"
+        anycast_ip_address: "10.0.0.0"
+        service: "DNS"
+        tags:
+          location: "{{ tag_value }}"
+        state: "present"
+      register: ac_config
+
+    - name: Get Information about the Anycast Configuration by ID
+      infoblox.universal_ddi.anycast_config_info:
+        id: "{{ ac_config.id }}"
+      register: ac_config_info
+    - assert:
+        that:
+         - ac_config_info.objects | length == 1
+         - ac_config_info.objects[0].name == ac_config.object.name
```

```suggestion
         - ac_config_info.objects[0].name == ac_config_name
```
Please make this change at other places as well

---

Done